### PR TITLE
DeleteReIp API call implementation + CG Tests

### PR DESCRIPTION
**Pull request by jimantz**
Implement the DeleteReIp API call in the ComputeAPI. Also provide unit tests for the Consistency Group accessor.

### CODE REVIEW

File: ComputeClient/Compute.Client.UnitTests/Drs/ConsistencyGroupAccessorTests.cs [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+
+namespace Compute.Client.UnitTests.Drs
+{
+	[TestClass]
+	public class ConsistencyGroupAccessorTests : BaseApiClientTestFixture
+	{
+		[TestMethod]
+		public async Task GetConsistencyGroups_ReturnsResponse()
+		{
+			requestsAndResponses.Add(ApiUris.MyAccount, RequestFileResponseType.AsGoodResponse("GetMyAccountDetails.xml"));
```
Comment (jamesoleary123) [inline ComputeClient/Compute.Client.UnitTests/Drs/ConsistencyGroupAccessorTests.cs]: Noticed those mocked calls for the GetMyAccountDetails API in a few tests - don't think its actually needed.

File: ComputeClient/Compute.Client.UnitTests/Drs/ConsistencyGroupAccessorTests.cs [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+			Assert.IsNotNull(consistencyGroupWithReIp.serverPair);
+			Assert.AreEqual(1, consistencyGroupWithReIp.serverPair.Count());
+			var serverPairWithReIp = consistencyGroupWithReIp.serverPair.ElementAt(0);
+			Assert.IsTrue(serverPairWithReIp.reIpAddressRulesDefined);
+		}
+
+		[TestMethod]
+		public async Task GetConsistencyGroups_PaginatedResponse()
+		{
+			PageableRequest pagingOptions = new DD.CBU.Compute.Api.Contracts.Requests.PageableRequest();
```
Comment (jamesoleary123) [inline ComputeClient/Compute.Client.UnitTests/Drs/ConsistencyGroupAccessorTests.cs]: very minor but think we can re-write this as 

```          
           var pagingOptions = new DD.CBU.Compute.Api.Contracts.Requests.PageableRequest
			{
				PageSize = 3,
				PageNumber = 1,
				Order = "sourceDataCenterId"
			};
```
Just a little bit more concise - reading around in c# think the steer is var should be used for local variables where its fairly obvious what the assignment will be based on the right side of the expression

File: ComputeClient/Compute.Client.UnitTests/Drs/ConsistencyGroupAccessorTests.cs [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+			Assert.IsNotNull(response);
+			Assert.AreEqual("CREATE_CONSISTENCY_GROUP", response.operation);
+			Assert.AreEqual("3389ffe8-c3fc-11e3-b29c-001517c4643e", response.info.ElementAt(0).value);
+		}
+
+		[TestMethod]
+		public async Task GetConsistencyGroupSnapshots_Returns()
+		{
+			Uri baseUri = ApiUris.GetConsistencyGroupSnapshots(accountId);
+			var paramUri = new Uri(String.Concat(baseUri.ToString(), "?consistencyGroupId=3389ffe8-c3fc-11e3-b29c-001517c4643e"), UriKind.Relative);
```
Comment (jamesoleary123) [inline ComputeClient/Compute.Client.UnitTests/Drs/ConsistencyGroupAccessorTests.cs]: this is useful for testing API filters from the compute api client 👍

**Review by jamesoleary123 — APPROVED** [review]

File: ComputeClient/Compute.Client.UnitTests/Drs/ConsistencyGroupAccessorTests.cs (hunk 1 above)
Comment (jimantz) [reply ComputeClient/Compute.Client.UnitTests/Drs/ConsistencyGroupAccessorTests.cs]: Nope, they're not. I've removed them now 👍

File: ComputeClient/Compute.Client.UnitTests/Drs/ConsistencyGroupAccessorTests.cs (hunk 2 above)
Comment (jimantz) [reply ComputeClient/Compute.Client.UnitTests/Drs/ConsistencyGroupAccessorTests.cs]: Yup, fixed both the pagingOptions and the var things.

**Pull request merged**